app/src/main/java/com/lightricks/feedexercise/database/Entity.kt

```diff
@@ -0,0 +1,14 @@
+package com.lightricks.feedexercise.database
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+/**
+ * todo: add Room's Entity data class(es) here
+ */
+@Entity(tableName = "feed_table")
+data class Entity(
+    @PrimaryKey
+    val id: String, // Kotlin note: "val" means read-only value.
+    val thumbnail_url: String?,
+    val is_premium: Boolean?)
```

Comment:
Naming convention: 
Kotlin code - `isPremium`
SQLite - `is_premium`.

Solution:
Add annotation` @ColumnInfo(name = "is_premium")`

Same for `thumbnailUrl`.